src/request/components/request-detail-panel-logging.tsx

```diff
@@ -0,0 +1,100 @@
+'use strict';
+
+import { ILogMessage } from '../messages/ILogMessage';
+import { ILoggingViewModel } from '../view-models/ILoggingViewModel';
+
+import React = require('react');
+
+interface ILogMessagesProps {
+    messages: ILogMessage[];
+}
+
+/**
+ * React class to display console messages
+ */
+class LogMessages extends React.Component<ILogMessagesProps, {}> {
```

Comment:
If it were re-used by other components or was unit tested independently, definitely.  If used only by this component, and being a view component where no unit tests are generally expected, then I'd tend to treat this like a "private class" in .NET.  It's there for the purposes of making the larger component easier to read, but is really just an implementation detail not meant to be visible outside that larger component.